pytest/tests/sanity/congestion_control.py

```diff
@@ -145,15 +135,15 @@ def __run_under_congestion(self, node):
             gas_used = chunk['header']['gas_used']
             congestion_info = chunk['header']['congestion_info']
 
-            self.assertEqual(int(congestion_info['buffered_receipts_gas']), 0)
             self.assertEqual(int(congestion_info['delayed_receipts_gas']), 0)
-            self.assertEqual(congestion_info['receipt_bytes'], 0)
 
             logger.info(
-                f"#{height} other gas used: {gas_used} congestion info {congestion_info}"
+                f"#{height} other  gas used: {gas_used} congestion info {congestion_info}"
             )
```

Comment:
nit: looks like an unintended change

Reply:
It's intended, it aligns it with the other log line above :D In the first one it's the "target" shard and here it's the "other" shard and "other" is one character shorter so I added an extra whitespace :)

Reply:
Ah okay, makes sense :+1: